dev/tests/static/testsuite/Magento/Test/Js/_files/eslint/.eslintrc-magento

```diff
@@ -68,14 +136,48 @@
             }
         ],
         "no-use-before-define": 2,
+        "no-useless-call": 2,
+        "no-useless-computed-key": 2,
+        "no-useless-constructor": 2,
+        "no-useless-escape": 2,
+        "no-useless-rename": 2,
+        "no-useless-return": 2,
+        "no-var": 2,
+        "no-void": 2,
+        "no-whitespace-before-property": 2,
         "no-with": 2,
```

Comment:
no-var may cause some errors since i see lot of js code still use var ?

Reply:
https://eslint.org/docs/rules/no-var

The idea here is to configure ESLint to encourage use of modern JavaScript. Yes, `var` still works but `let` and `const` are preferred.